lib/vtls/schannel.c

```diff
@@ -1716,4 +1903,69 @@ static CURLcode verify_certificate(struct connectdata *conn, int sockindex)
 }
 #endif /* _WIN32_WCE */
 
+static void Curl_schannel_checksum(const unsigned char *input,
```

Comment:
I guess it can't?  It's used to define curlssl_sha256sum (and later curlssl_md5sum), and that function doesn't have a return value.  schannel appears to be the only backend where it even has a possibility of failure, which might be why it wasn't implemented before...